dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -699,7 +696,6 @@ Resources:
   VPCNatGateway:
     Condition: DeployAGwithVPC
     Type: "AWS::EC2::NatGateway"
-    DependsOn: VPCNatGatewayElasticIP
```

Comment:
Why can we remove it now?

Reply:
!GetAtt in the next lines already implies a dependency on VPCNatGatewayElasticIP

Reply:
Ahh, right, so it was superfluous previously already, not only after your changes in this PR